include/unifex/receiver_concepts.hpp

```diff
@@ -140,13 +178,15 @@ namespace _rec_cpo {
 } // namespace _rec_cpo
 
 using _rec_cpo::set_value;
+using _rec_cpo::set_next;
 using _rec_cpo::set_error;
 using _rec_cpo::set_done;
 
 template <typename T>
 constexpr bool is_receiver_cpo_v = is_one_of_v<
```

Comment:
I agree that set_next belongs in a separate trait

Reply:
perhaps we need a constexpr cpo to customize what gets forwarded and use the qualifiers on T in the calculation. some other way to package teh qualifiers would be nine since T will create many instantiations of forward_by_default.

`forward<CPO, const T&>()`